### DISCUSSION TITLE
Can you recommend a camera?

### DISCUSSION THREAD

**Discussion by gherlein** [bluenviron/mediamtx#905] [Questions]
Can anyone recommend a camera they like that works with this software well?  HD, ideally 30fps.  Reliable.  Prefer wired, prefer PoE.  Or alternately, WiFi powered by wall wart.  This is mostly for playing, but if I am successful in my testing I want to be able to buy 5+ so I prefer a dependable supplier.  Quality matters, but so does price.

I have Unifi cameras now and they are a pain in a lot of ways.  Not saying I'm replacing them, but I could really use something better.

I'm a developer and long-time linux user.  I don't want to build a kit.  I want to buy a camera, with an enclosure, that will work in humid/hot weather.  I want to do some software development around it.

What are you using that you like?

**Comment by aler9** [comment]
Hi, the problem of choosing a camera for playing is that choices are limited by budget. I generally test this server with professional cameras that were bought for a specific and financed project. These are chosen by looking at the project specification, whether they will be mounted on a car, on a UAV, outside or inside. They can be automotive-level, water resistant (IP rating), with a wide optical zoom, etc... they can also be fitted with computational power and host some algorithms.

Manufactures are mainly HikVision, Axis, Avigilon. Between the 3, the cheapest is HikVision.

BTW, i'm still a fan of the Raspberry Pi camera. with 35€ you can buy something with a quality that is comparable to the one of a professional camera, and much smaller. I've brought it with me during some presentations in order to test video AI algorithms on the audience, and it never failed. Therefore i'd probably stick with it, even if you need to build the enclosure :)

[a comment by an automated account is not shown]
